README.md

```diff
@@ -54,7 +54,7 @@ Let's say you've been working on a feature for a couple days, most likely there
 
 ```
 $ git checkout development
-$ git pull upstream development
+$ git pull upstream development or $git pull
```

Comment:
Please remove `or $git pull`. If you don't set the the remote, it will default to `origin`, meaning it will pull from your fork instead of the original repo.

Also, follow these instructions to rebase your branch and get the latest changes ;)